### PR TITLE
Add section for Office PIA consumption sample

### PR DESCRIPTION
**Pull request by AaronRobinsonMSFT**
## Summary

Add a section for the Office PIA consumption sample.

See dotnet/samples#487

cc @mairaw @jeffschwMSFT @richlander

### CODE REVIEW

File: docs/samples-and-tutorials/index.md [hunk 1]
```diff
@@ -56,6 +56,10 @@ This sample demonstrates many of the problems that can be solved by using Expres
 
 This series of samples demonstrate many of the features of Language Integrated Query (LINQ). The [completed sample](https://github.com/dotnet/samples/tree/master/core/linq/csharp) is available in the dotnet/samples repository on GitHub.
 
+**Microsoft Office PIA Sample**
+
+This [sample](https://github.com/dotnet/samples/tree/master/core/extensions/ExcelDemo) demonstrates the consumption of [Microsoft Office PIAs](https://docs.microsoft.com/visualstudio/vsto/office-primary-interop-assemblies?view=vs-2017) in .NET Core.
```
Comment (mairaw) [inline docs/samples-and-tutorials/index.md]: Suggestion:
 
```suggestion
The [ExcelDemo](https://github.com/dotnet/samples/tree/master/core/extensions/ExcelDemo) sample demonstrates the consumption of [Microsoft Office PIAs](/visualstudio/vsto/office-primary-interop-assemblies) in .NET Core.
```

**Review by mairaw — APPROVED** [review]
LGTM. Left a suggestion for you to consider so the link work in offline books too inside VS.

File: docs/samples-and-tutorials/index.md (hunk 1 above)
Comment (mairaw) [reply docs/samples-and-tutorials/index.md]: Also, should we state this is using .NET Core 3?

Comment (AaronRobinsonMSFT) [reply docs/samples-and-tutorials/index.md]: I am not against that. My desire was to keep those details in the project, but if it makes more sense to put that here sure. Opinion?

**Comment by AaronRobinsonMSFT** [comment]
@mairaw Looks like we got out first inquiry in this area 😄 Are we ready to merge this?

See dotnet/core#2104

**Comment by mairaw** [comment]
Sure! Publishing will happen tonight.

**Pull request merged**